_o_faq_posts/08.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: single
+title: "How can I configure environments for my application?"
+body: "
+An Angular Application Environment is JSON configuration information that tells the build system which files to change when you use ng build and ng serve.<br/>
+You can also create variables with different values in each of the environment files.<br/>
+Check <a href='https://medium.com/@balramchavan/configure-and-build-angular-application-for-different-environments-7e94a3c0af23' target='blank'>this</a> to understand more about the environment configuration.
```

Comment:
Pointing to medium articles maybe can bring us problems in the future if the artricle is deleted.